site/app/controllers/SelfRejoinController.php

```diff
@@ -129,6 +129,14 @@ private function sendRejoinedStudentEmail($joined_section): void {
             $email = new Email($this->core, $details);
             array_push($emails, $email);
         }
+        $sysadamin_email = $this->core->getConfig()->getSysAdminEmail();
+        if (!empty($sysadamin_email)) {
+            unset($details["to_user_id"]);
+            $details["email_address"] = $sysadamin_email;
+            $details["to_name"] = "Sysadmin";
+            $email = new Email($this->core, $details);
+            array_push($emails, $email);
```

Comment:
The array push shorthand is generally cleaner:
```suggestion
            $emails[] = $email;
```

Reply:
I prefer the other syntax, but I understand the latter is more PHP-esque, so I'll use what you recommend.